src/context/loadIsoVolumes.tsx

```diff
@@ -20,6 +25,15 @@ export const loadIsoVolumes = async (
 
 export const loadVolumes = async (
   project: string,
+  hasStorageVolumesAll: boolean,
+): Promise<LxdStorageVolume[]> => {
+  return hasStorageVolumesAll
+    ? fetchAllStorageVolumes(project)
+    : await collectAllStorageVolumes(project);
```

Comment:
this `await` isn't needed since `loadVolumes` is awaited when its called.

Reply:
good catch, removed it.